app.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+module.exports = {
+  englishToPigLatin: function(word) {
+    //word verification
+    if (typeof word !== 'string' || Number(word) / 2) {
+      console.log('Please enter a valid word.');
+      return false;
+    }
+
+    //multi word verification
+    if (word.split('').indexOf(' ') !== -1) {
+      console.log('Please input a single word.');
+      return false;
+    }
+
+    let firstLetterVowel = 'AEIOUaeiou';
+    let appendage = 'ay';
+    let firstLetter = word.slice(0, 1);
+
+    if (firstLetterVowel.indexOf(firstLetter) === -1) {
```

Comment:
I don't think this will handle words like "think" or any words that have more than one consonant in front before it reaches a vowel.